content/en/platform/corda/4.12/community/node-upgrade-notes.md

```diff
@@ -74,7 +74,7 @@ Make sure it’s available on your path, and that you’ve read the [Release not
 node requires.
 
 {{< important >}}
-Corda 4 requires Java 8u171 or any higher Java 8 patch level. Java 9+ is not currently supported.
+Corda 4 requires Java 17.
```

Comment:
### for this line:

> Corda 4 requires Java 17.0.9 or any higher Java 17 patch level. Java 18+ is not currently supported.

### for this file:

Please can you also add the changes to this file from this PR:
https://github.com/corda/corda-docs-portal/pull/2176/files#diff-415c0ea577a3609930e1405747ef6e3f7c2b129e6c5299345c92befef84c7290

There is a bit of re-jigging of steps